main.go

```diff
@@ -66,47 +78,106 @@ func preLaunch() {
 	usbarmory.LED("white", false)
 }
 
+// read reads the firmware bundle for the trusted OS from internal storage, the
+// OS proof bundle is *not* verified by this function.
+func read(card *usdhc.USDHC) (fw *firmware.Bundle, err error) {
+	blockSize := card.Info().BlockSize
+	if blockSize != expectedBlockSize {
+		return nil, fmt.Errorf("h/w invariant error - expected MMC blocksize %d, found %d", expectedBlockSize, blockSize)
```

Comment:
Where you would write “actual” and “expected”, prefer using the words “got” and “want”, respectively.
https://google.github.io/styleguide/go/decisions.html#got-before-want